.github/workflows/ci-postgres-mysql.yml

```diff
@@ -106,9 +106,9 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.1
       - run: corepack enable
-      - uses: actions/setup-node@v4.0.1
+      - uses: actions/setup-node@v4.0.2
         with:
-          node-version: 18.x
+          node-version: 20.x
```

Comment:
Can we extract and reuse this step so we have a single spot to update?

Reply:
I want to do that in a separate PR. [Started a while ago, but ran into some issues](https://github.com/netroy/n8n/commit/0364b079dc5f17ee0a2fdf5ab84451911904a494).